### ISSUE TITLE
Problem with the target folder and the ContextQuickie.TeamProvider

### ISSUE THREAD

**Issue by oilid** [ContextQuickie/ContextQuickie#25]
Hey Roland,

today I got the following  exception with the ContextQuickie.TeamProvider during an incrementell build of eclipse:

```
ContextQuickie.TeamProvider
Error during applying changes to working copy
org.apache.subversion.javahl.ClientException: Entry already exists
svn: Path 'C:\projects\foo-trunk\work\foo-app-aggregator\foo-app\target\test-classes\de\xxx\yyyy\zzz\gggg\po\BeantragungHelperTest.xls' already exists

	at org.apache.subversion.javahl.SVNClient.copy(Native Method)
	at contextquickie.teamprovider.svn.ResourceDeltaVisitor$2.run(ResourceDeltaVisitor.java:197)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:56)
```

But I have no real hint why this happenend. I only know that the "target" folder is on svn ingnore...
Is there maybe a bug in the ContextQuickie.TeamProvider with the svn ignore properties? Because I saw sometimes that our projects lost the svn ingnore information of the target (build) folder...

Greetings
Oilid

**Comment by ContextQuickie** [comment]
Hi @oilid ,
maybe there is an issue in the plugin. As far as I see it happened during a copy operation. Currently I'm only checking if the source of the operation is under version control but not if the target folder is already under version control.
BR
Roland

**Comment by ContextQuickie** [comment]
Hello @oilid ,
this issue should be fixed in version 0.11.1
BR
Roland
